### PR TITLE
Gitpod support

### PR DESCRIPTION
**Pull request by CuriouzK0d3r**
### Description of PR

What component(s) does this PR affect?

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [x] Build System

Does the PR require changes on other components? If yes, please mark the components:

- [ ] Back-End (Database, Detection/Configuration/etc. Microservices)
- [ ] Front-End (Flask, API, UI, etc)
- [ ] Monitor (RIPE RIS, BGPStream RV/RIS/CAIDA, etc.)
- [ ] Docs
- [ ] Build System

### Related Issue

Resolves #

### Type

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update
- [ ] None of the above

### Checklist:

- [ ] I have read the **[contributing guide](https://github.com/FORTH-ICS-INSPIRE/artemis/blob/master/CONTRIBUTING.md)** and my code conforms to the guidelines.
- [ ] This change requires a change in the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -137,6 +137,20 @@ start:
...
+gitpod-start: # start local setup
+gitpod-start:
+	@if [ ! -d "local_configs" ]; then \
+		mkdir -p local_configs && \
+		mkdir -p local_configs/backend && \
+		mkdir -p local_configs/monitor && \
+		mkdir -p local_configs/frontend && \
+		cp -rn backend-services/configs/* local_configs/backend && \
+		cp -rn monitor-services/configs/* local_configs/monitor && \
+		cp other/frontend/configs/nginx-gitpod.conf local_configs/frontend/nginx.conf; \
```
Comment (vkotronis) [inline Makefile:150]: The correct command is:
```
mkdir -p local_configs && \
mkdir -p local_configs/backend && \
mkdir -p local_configs/monitor && \
mkdir -p local_configs/frontend && \
cp -rn backend-services/configs/* local_configs/backend && \
cp backend-services/configs/redis.conf local_configs/backend/redis.conf && \
cp -rn monitor-services/configs/* local_configs/monitor && \
cp -rn other/frontend/configs/* local_configs/frontend
```

Comment (vkotronis) [reply Makefile:150]: so you simply need to add the redis conf so that it works as expected. Otherwise the PR is fine.

File: Makefile [hunk 2]
```diff
@@ -137,6 +137,21 @@ start:
...
+gitpod-start:
+	@if [ ! -d "local_configs" ]; then \
+		mkdir -p local_configs && \
+		mkdir -p local_configs/backend && \
+		mkdir -p local_configs/monitor && \
+		mkdir -p local_configs/frontend && \
+		cp -rn backend-services/configs/* local_configs/backend && \
+		cp backend-services/configs/redis.conf local_configs/backend/redis.conf && \
+		cp -rn monitor-services/configs/* local_configs/monitor && \
+		cp other/frontend/configs/nginx-gitpod.conf local_configs/frontend/nginx.conf
```
Comment (vkotronis) [inline Makefile:151]: don't we need a semicolon here?

Comment (CuriouzK0d3r) [reply Makefile]: added

**Review by vkotronis — APPROVED** [review]
LGTM, thanks Alex!

**Pull request merged**